aviary/interface/methods_for_level1.py

```diff
@@ -185,6 +185,9 @@ def _exec_level1(args, user_args):
     if isinstance(args.input_deck, list):
         args.input_deck = args.input_deck[0]
 
+    file_name_stem = Path(args.input_deck).stem
+    args.outdir = args.outdir + os.sep + file_name_stem
```

Comment:
Does this always overwrite an existing `outdir` if the user supplies it? @crecine does this change your expected behavior?

Reply:
You are right. I should consider the scenario that the user provide a special `outdir`. Now, I added a logic that check whether `outdir` is the default directory. If it is, add input deck name to it.